Tests/MessagingInApp/Views/InAppMessageViewTest.swift

```diff
@@ -561,16 +580,12 @@ class InAppMessageViewTest: IntegrationTest {
 
 @MainActor
 extension InAppMessageViewTest {
-    func onCustomActionButtonPressed(onInlineView inlineView: InAppMessageView) {
-        // Triggering the custom action button on inline message from the web engine
-        // mocks the user tap on custom action button
-        getWebEngineForInlineView(inlineView)?.delegate?.tap(name: "", action: "Test", system: false)
-    }
+    func setupQueueMock() -> MessageQueueManagerMock {
```

Comment:
Some test functions still benefit from mocking the `MessageQueueManager`. Added this util function to easily do that even though the test class by default does not mock the manager.